src/test/surveyBuilder/nodeDefBuilder.ts

```diff
@@ -0,0 +1,32 @@
+import { Survey } from '../../survey'
+import { NodeDef, NodeDefFactory, NodeDefEntity, NodeDefProps, NodeDefPropsAdvanced, NodeDefType } from '../../nodeDef'
+
+export abstract class NodeDefBuilder {
+  props: NodeDefProps
+  propsAdvanced: NodeDefPropsAdvanced
+  type: NodeDefType
+
+  constructor(name: string, type: NodeDefType) {
+    this.type = type
+    this.props = {
+      name,
+    }
+    this.propsAdvanced = {}
+  }
+
+  _createNodeDef(params: { nodeDefParent?: NodeDefEntity } = {}): NodeDef<NodeDefType, NodeDefProps> {
```

Comment:
no need to use _ prefix. make the method protected

Reply:
ok